pkg/querier/queryrange/roundtrip.go

```diff
@@ -132,46 +132,51 @@ func NewTripperware(
 		}
 	}
 
-	indexStatsTripperware, err := NewIndexStatsTripperware(cfg, log, limits, schema, LokiCodec, statsCache,
+	var codec queryrangebase.Codec = DefaultCodec
+	if cfg.AcceptedQueryResponseFormat == "protobuf" {
```

Comment:
Can we not detect this automatically?

Reply:
We do. This is for requesting the protobuf from the querier by the frontend even if the frontend received a request for JSON.

Reply:
Am I missing something, won't queries from external sources always be returned a JSON response? So this line is required regardless of what the external results format should be, this sets the format we want to use internally between services.

Reply:
> So this line is required regardless of what the external results format should be, this sets the format we want to use internally between services.

Yes. However, we don't have to set it if we want JSON because we default to JSON.